std_msgs/Header header
int32 state

int32 DISABLED          = 0
int32 ARMED             = 1
int32 DISABLED_COMPLETE = 2